.gitignore

```diff
@@ -221,3 +221,5 @@ FakesAssemblies/
 **/*.Server/GeneratedArtifacts
 **/*.Server/ModelManifest.xml
 _Pvt_Extensions
+
+somefuck
```

Comment:
?

Reply:
github not build if i doing force push :) i will squash all after changes

Reply:
All questions fixed